src/main/java/codeu/controller/LoginServlet.java

```diff
@@ -30,6 +30,8 @@
   /** Store class that gives access to Users. */
   private UserStore userStore;
 
+
```

Comment:
couple extra newlines here

Reply:
Done!

Reply:
it looks like it's still here :)

Reply:
Sorry thought you meant to add extra newlines but now removed them